lib/hammer_cli_katello/flatpak_remote_repository.rb

```diff
@@ -0,0 +1,53 @@
+module HammerCLIKatello
+  class FlatpakRemoteRepository < HammerCLIKatello::Command
+    resource :flatpak_remote_repositories
+    command_name 'remote-repository'
+    desc _('View and manage flatpak remote repositories')
+
+    class ListCommand < HammerCLIKatello::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :label, _("Label")
+      end
+      build_options do |o|
+        o.expand(:all).including(:flatpak_remotes, :organizations)
+      end
+    end
+
+    class InfoCommand < HammerCLIKatello::InfoCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :label, _("Label")
+        from :flatpak_remote do
+          field :id, _("Flatpak Remote ID")
+          field :name, _("Flatpak Remote Name")
+          field :url, _("Flatpak Remote URL")
+        end
+        collection :manifests, _("Manifests"), hide_blank: true, hide_empty: true do
+          field :name, _("Manifest Name")
+          field :digest, _("Manifest Digest")
+          field :tags, _("Manifest tags")
+        end
+      end
+      build_options do |o|
+        o.expand(:all).including(:flatpak_remotes, :organizations)
+      end
+    end
+
+    class MirrorCommand < HammerCLIKatello::SingleResourceCommand
```

Comment:
Perhaps unrelated to this PR, but do you know what "organization title" is? 

We typically only have ID / Name / Label...

```
Usage:
    hammer flatpak-remote remote-repository mirror [OPTIONS]

Options:
 --async                                        Do not wait for the task
 --flatpak-remote[-id] VALUE/NUMBER             Name/Id of associated flatpak remote
 --id NUMBER                                    Flatpak remote repository id
 --name VALUE                                   Name to search by
 --organization[-id|-title|-label] VALUE/NUMBER Name/Title/Label/Id of associated organization
 --product[-id] VALUE/NUMBER                    Product name/id to mirror the remote repository to
 -h, --help                                     Print help
```

Reply:
```
[2] pry(main)> org.title
=> "Default Organization"
[3] pry(main)> org.name
=> "Default Organization"
[4] pry(main)> org.label
=> "Default_Organization"
```

Title seems to be the same as name.

Reply:
Interesting!